react/transforms/address.ts

```diff
@@ -62,8 +62,25 @@ export function removeValidation(address) {
 export function addNewField<FieldName extends keyof ValidatedField>(
   address: AddressWithValidation,
   fieldName: FieldName,
-  value: ValidatedField[FieldName]
+  value: ValidatedField[FieldName] |  ((fieldValue: ValidatedField) => boolean)
 ): AddressWithValidation {
+  
+  if(value instanceof Function){
```

Comment:
can you run prettier on the repo? just run `yarn lint --fix` at the root and you should be good to go :-)

Reply:
also, I think I prefer using `typeof` instead of `instanceof` here, you can find more details on [this StackOverflow answer](https://stackoverflow.com/questions/899574/what-is-the-difference-between-typeof-and-instanceof-and-when-should-one-be-used)
```suggestion
  if (typeof value === 'function') {
```